blogposts/2020/software-engineer-career-paths.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Software Engineer Career Paths"
```

Comment:
```suggestion
title: "Software engineer career paths"
```

https://about.sourcegraph.com/handbook/communication/content_guidelines/actionable_language#headings-and-subheadings